src/utils/natspec.ts

```diff
@@ -14,7 +14,7 @@ export function* extractNatspec(node: {
     typeof node.documentation === 'string' ? node.documentation : node.documentation?.text ?? '';
 
   for (const { groups } of execall(
-    /^\s*(?:@(?<title>\w+)(?::(?<tag>[a-z][a-z-]*))? )?(?<args>(?:(?!^\s@\w+)[^])*)/m,
+    /^\s*(?:@(?<title>\w+)(?::(?<tag>[a-z][a-z-]*))?)?(?: (?<args>(?:(?!^\s@\w+)[^])*))?/m,
```

Comment:
FYI, the issue here is that the space in the tag block prevents parsing title+tag without a space/args at the end.